src/main/java/io/mbab/sda/groupproject/entity/Team.java

```diff
@@ -27,6 +28,10 @@
     @Column(length = 128, columnDefinition = "int default 0")
```

Comment:
usun length, jezeli default 0 na bazie to nie potrzebny Wrapper uzyj małego double